### ISSUE TITLE
Timestamp still shows in man documentation

### ISSUE THREAD

**Issue by speedingdaemon** [spf13/cobra#741]
I thought that this issue was resolved with https://github.com/spf13/cobra/issues/142
But even after specifying in my root cmd:
        DisableAutoGenTag: true,

I see this text in my man documentation:
*Auto generated by spf13/cobra on 6-Sep-2018*

markdown and ReST documentation don't have this text. 
Am I doing something wrong?

Here is my code:
var rootCmd = &cobra.Command{
        Use:               "hello",
        Short:             "test",
        Long:              "my test",
        DisableAutoGenTag: true,
}

func genManDocs() {
        header := &doc.GenManHeader{
                Title:   "MINE",
                Section: "3",
        }
        err := doc.GenManTree(rootCmd, header, "./")

        if err != nil {
                fmt.Println(err)
        }
}

**Comment by Skarlso** [comment]
Yep, second that. It's still in there.

**Comment by Simran-B** [comment]
I want to add that it's also showing in the Markdown documentation, so likely a general issue of `DisableAutoGenTag` not working properly (not passed on correctly?)

**Comment by kamalgrover** [comment]
Guys, Is there a workarround for the same ?

**Comment by Skarlso** [comment]
Not that I know off, sorry. :/

**Comment by kamalgrover** [comment]
I used the "source" member to overcome this.

**Comment by kamalgrover** [comment]
Here is the code snippet.
`func main() {

	header := &doc.GenManHeader{
		Title:   "shorthand",
		Section: "1",
		Manual:  "My Product Manual",
		Source: "Copyright (C) 2020 All Rights Reserved.",

	}
	//Disables the autogenerated comment at the end of the man page.
	cmd.RootCmd.DisableAutoGenTag = true
	err := doc.GenManTree(cmd.RootCmd, header, "doc/")
	if err != nil {
		log.Fatal(err)
	}
}`

**Comment by jpmcb** [comment]
@jharshman I did some investigation on this and seems that `cmd.DisableAutoGenTag` will only skip placing the auto gen tag in the body while the header "source" will get auto populated with the auto gen tag if nothing is specified.

I was able to work around this by specifying something other than an empty string:
```
header := &doc.GenManHeader{
	Title: "someTitle",
	Section: "1",
	Source: " ",
}
```

Would we expect setting `cmd.DisableAutoGenTag = true` to disable the string "Auto generated by spf13/cobra" _everywhere_ when docs are generated? If so, I'd recommend we extend this to disable _all_ auto generated tags when the bool is set to true. I can work on this as well if that's the direction we want to go.
